README.md

```diff
@@ -20,7 +20,7 @@ The latest stable release of the NUnit Console is [available on NuGet](https://w
 
 The Console/Engine are available in various packages:
 - [NUnit.ConsoleRunner](https://www.nuget.org/packages/NUnit.ConsoleRunner/): The NUnit Console, with no extensions.
-- [NUnit.Console](https://www.nuget.org/packages/NUnit.Console/): The NUnit Console, with key extensions additionally packaged. Also available as an [msi installer](https://github.com/nunit/nunit-console/releases).
+- [NUnit.Console](https://www.nuget.org/packages/NUnit.Console/): The NUnit Console, with key extensions additionally packaged. Also available as an [msi installer](https://github.com/nunit/nunit-console/releases), remember to add its install location to your environment variable after installing, such as `C:\Program Files (x86)\NUnit.org\nunit-console`.
```

Comment:
I think the original is trying to say to much in a single bullet-point and adding the tip demonstrates that. Nuget packages don't want to have anything added to the path, clearly, only the msi.

What if we change line 21 to say "various NuGet packages"? Then we could put in a new paragraph for the msi, like "An msi installer is also available if you want to install the runner in a central location. You may want to add the location to the path after installing..."

Should we also deal with the fact that the chocolatey packages have different names from the nuget packages?

Reply:
OK, I got it. Thanks for your reply

Reply:
So, I close this PR?

Reply:
Only if you don't want to improve it. For me, it should at least say which environment variable to change if you install the msi. The other changes would improve it IMO but aren't essential.

To update a PR you just push further changes to your branch on which it is based.

Reply:
Hi CharliePoole, I have updated the README, can you please take some time to review it?

Thanks.